recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/xen/xen_git.bbappend

```diff
@@ -0,0 +1,28 @@
+################################################################################
+# We only need Xen tools, so we can start domains
+################################################################################
+XEN_REL = "4.9"
+
+SRCREV = "${AUTOREV}"
+
+SRC_URI = "git://github.com/xen-troops/xen.git;protocol=https;branch=vgpu-dev"
```

Comment:
You are comiting this into `master` branch for generic `xt-prod` product, but using `vgpu-dev` branch of XEN. This is confusing.

Reply:
this is CES2018 product, won't fix